samples/aws_lambda/lazy_aws_lambda.py

```diff
@@ -0,0 +1,59 @@
+# ------------------------------------------------
+# instead of slack_bolt in requirements.txt
+import sys
+import time
+
+sys.path.insert(1, "vendor")
+# ------------------------------------------------
+
+import logging
+
+from slack_bolt import App
+from slack_bolt.adapter.aws_lambda import SlackRequestHandler
+
+# process_before_response must be True when running on FaaS
+app = App(process_before_response=True)
+
+
+@app.middleware  # or app.use(log_request)
+def log_request(logger, payload, next):
+    logger.debug(payload)
+    return next()
+
+
+command = "/hello-bolt-python-lambda"
+
+
+def respond_to_slack_within_3_seconds(payload, ack):
+    if payload.get("text", None) is None:
+        ack(f":x: Usage: {command} (description here)")
+    else:
+        title = payload["text"]
+        ack(f"Accepted! (task: {title})")
+
+
+def process_request(respond, payload):
+    time.sleep(5)
+    title = payload["text"]
+    respond(f"Completed! (task: {title})")
+
+
+app.command(command)(
+    ack=respond_to_slack_within_3_seconds,
+    lazy=[process_request]
```

Comment:
The `process_request` method is executed in another Lambda invocation (the same Lambda function). The duplicated request is marked as `lazy_only: True` and the request contains the name `process_request` in it.